pkg/ocm/ocm.go

```diff
@@ -23,24 +22,26 @@ var (
 
 type ocmClient struct {
 	conn *sdk.Connection
-	cfg  *sdkcfg.Config
 }
 
 // New will create a new ocm client by using the path to a config file
 // if no path is provided, it will assume it in the default path
 func New(ocmConfigFile string) (ocmClient, error) {
+	var err error
 	client := ocmClient{}
-	cfg, err := newConfigFromFile(ocmConfigFile)
-	if err != nil {
-		return client, fmt.Errorf("failed to load config file: %w", err)
+
+	if ocmConfigFile != "" {
+		err := os.Setenv("OCM_CONFIG", ocmConfigFile)
+		if err != nil {
+			return client, err
+		}
 	}
-	client.cfg = cfg
 
-	conn, err := client.cfg.Connection()
+	client.conn, err = sdk.NewConnectionBuilder().Load(ocmConfigFile).Build()
 	if err != nil {
-		return client, fmt.Errorf("can't create connection: %w", err)
+		return client, fmt.Errorf("failed to load config file: %w", err)
```

Comment:
Do we want to change this error message? Failing to load the config is just one possible errors.. others can be problems with establishing the connection and more.

Reply:
I think a more generic error here would be better for sure.